### PR TITLE
Server Side TLS 5.0

### LINKED ISSUE #217: It is time to add support for TLS 1.3
TLS 1.3 is now in release in both Chrome and Firefox and supported by Google and Cloudflare.

### PR DESCRIPTION
**Pull request by april**
Fixes #217, #211, #191, and #178.

### CODE REVIEW

File: Server_Side_TLS.mediawiki [hunk 1]
```diff
@@ -1,600 +1,236 @@
...
-
-The ordering of a ciphersuite is very important because it decides which algorithms are going to be selected in priority. Each level shows the list of algorithms returned by its ciphersuite. If you have to pick ciphers manually for your application, make sure you keep the ordering.
-
-The ciphersuite numbers listed come from the IANA [https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-4 TLS Cipher Suite Registry]. Previous versions of these recommendations included draft numbers for ECDHE-ECDSA-CHACHA20-POLY1305 (0xCC,0x14) and ECDHE-RSA-CHACHA20-POLY1305 (0xCC,0x13).
+<p style="max-width: 60em;">OpenSSL will ignore cipher suites it doesn't understand, so always use the full set of cipher suites below, in their recommended order. The use of the <span style="color: gray; font-weight: bold;">Old</span> configuration with modern versions of OpenSSL may require custom builds with support for SSLv3 and deprecated ciphers.</p>
```
Comment (nmxcgeo) [inline Server_Side_TLS.mediawiki]: Why do you mention enabling SSLv3 via custom builds, when SSLv2 and SSLv3 will be abandoned by this guide after this PR is merged? However a custom OpenSSL build for TLSv1.0 and TLSv1.1 might be necessary (e.g. for upcoming Debian 10).

Comment (april) [reply Server_Side_TLS.mediawiki]: Because you have to enable SSLv3 in the build to get access to 3DES.  :)

Comment (april) [reply Server_Side_TLS.mediawiki]: That said, I can see how that would be confusing, so I'll trim it up a little bit.

**Comment by april** [comment]
Thanks for everyone's hard work on this. It's been a long journey, but we finally got there.  :)

**Pull request merged**